### ISSUE TITLE
Can't get Augur filters working

### ISSUE THREAD

**Issue by kevinday** [AugurProject/augur.js#99]
Testing with `1.5.4`, I'm not seeing any events for marketCreated or price. In one console I'm doing
```
var options = {
     http: "http://localhost:8545",
     ws: null
};
var augur = require("augur.js");
augur.connect(options);
augur.filters.listen({
    marketCreated: function (msg) {
        console.log("market created filter:", msg);
    },
    price: function (msg){
    	console.log("price filter:", msg);
    }
}, function (filters) {
    console.log("setup complete:", filters);
});
```
And in the other, I'm creating a market:
```
augur.createSingleEventMarket({
    branchId: augur.constants.DEFAULT_BRANCH_ID,
    description: "Will the temperature in Austin, TX be > 90F on July 1, 2021?",
    expDate: new Date("7/2/2021").getTime() / 1000,
    minValue: 1,
    maxValue: 2,
    numOutcomes: 2,
    makerFee: ".002",
    takerFee: ".05",
    tags: ["weather", "temperature", "climate change"],
    extraInfo: "Hello world!  Are you getting hotter?",
    resolution: "generic",
    onSent: function (sentResponse) {console.log("MarketSentResponse:", sentResponse);},
    onSuccess: function (successResponse) { console.log("MarketSuccessResponse:", successResponse); },
    onFailed: function (failedResponse) { console.log("MarketFailedResponse:", failedResponse);}
});
```
Market creation is successful, but no filters are coming through. I've tried the same thing with both `augur.buy` and `augur.trade`, and am not seeing price filters for either of those.

Does this test look valid to you?

**Comment by tinybike** [comment]
Your test looks fine.  I just forgot to update the logs in `src/constants.js` when I made the last round of changes -- need to add that to the automatic API update script!

Also, it looks like the `marketCreated` and `tradingFeeUpdated` events accidentally got deleted during the latest augur-core mega-merge.  Adding them back in now...

**Comment by tinybike** [comment]
Fixed in 3ed18f0118f9e4c96455e21443a838e58a36aab3 (augur.js@1.5.8).
